### PR TITLE
Fix docs in References/Samples

### PR DESCRIPTION
**Pull request by ruthryi**
- Fix https://github.com/wso2/docs-apim/issues/1711
- Fix links for 3.2.0
- Formatting

### CODE REVIEW

File: en/docs/reference/samples/api-development-sample/collaborative-api-development-sample.md [hunk 1]
```diff
@@ -58,14 +50,14 @@ The following are the user credentials that you need to use when signing in to t
 
 Follow the instructions below to implement the latter mentioned business scenario using WSO2 API Manger.
 
-1.  Create three [different tenants](https://docs.wso2.com/display/AM260/Managing+Tenants) ( `finance.abc.com, core.abc.com,` and `operations.abc.com` ) for the three departments, with [users](https://docs.wso2.com/display/AM260/Key+Concepts#KeyConcepts-Usersandroles) (John, Tom, and Bob respectively) who can [create APIs](https://docs.wso2.com/display/AM260/Create+and+Publish+an+API) .
+1.  Create three [different tenants]({{base_path}}/administer/multitenancy/managing-tenants) ( `finance.abc.com, core.abc.com,` and `operations.abc.com` ) for the three departments, with [users]({{base_path}}/administer/managing-users-and-roles/introduction-to-user-management) (John, Tom, and Bob respectively) who can [create APIs]({{base_path}}/getting-started/quick-start-guide/#step-1-create-and-publish-an-api) .
```
Comment (Mariangela) [inline en/docs/reference/samples/api-development-sample/collaborative-api-development-sample.md:53]: Remove the space between the bracket and ` please. Check the rest of the doc for this issue.
( `finance.abc.com,

File: en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md [hunk 2]
```diff
@@ -36,30 +34,29 @@ In the future, ABC bank will need more and more functionality exposed through AP
...
+### API developer optimizing the API output by identifying the client
+
+API developers can use the headers sent by the client to detect the type of the client (e.g., if the client is a mobile device or not). For example, consider a client who is accessing ABC bank's API’s and performing online account balance checking. In the latter mentioned scenario, if the API developer needs to allow mobile device clients to only access a selected set of data of their account details, and expose all the data to desktop application clients, this can be achieved when using WSO2 API Manager by having a custom sequence in the API definition. For more information on adding custom sequences, see [Adding Mediation Extensions]({{base_path}}/learn/api-gateway/message-mediation/adding-a-class-mediator) .
```
Comment (Mariangela) [inline en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md:55]: There should not be a space before the fullstop.

.......adding-a-class-mediator) .

File: en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md [hunk 3]
```diff
@@ -70,21 +67,22 @@ For example if the API developer needs to restrict sending the address element o
...
 
 The API developer can add the following custom sequences to the API definition to achieve the latter mentioned scenario.
-In the following custom IN-sequence, it reads the User-Agent header and stores it in a custom property in WSO2 API Manager.
 
 **In-sequence**
 
+In the following custom IN-sequence, it reads the User-Agent header and stores it in a custom property in WSO2 API Manager.
+
 ``` java
     <sequence xmlns="http://ws.apache.org/ns/synapse" name="custom-in">
```
Comment (Mariangela) [inline en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md:76]: The first indentation of the code should start inline with ```
Please fix this in other places as well

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/reference/samples/api-development-sample/collaborative-api-development-sample.md (hunk 1 above)
Comment (ruthryi) [reply en/docs/reference/samples/api-development-sample/collaborative-api-development-sample.md]: Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/cb2834840dadcffdece6e356763992c34025e896

File: en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md (hunk 2 above)
Comment (ruthryi) [reply en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md]: Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/cb2834840dadcffdece6e356763992c34025e896

File: en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md (hunk 3 above)
Comment (ruthryi) [reply en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md]: Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/25c0d6d698df4049f8d4a78b186ae4377dfe9977

**Review by Mariangela — APPROVED** [review]

**Pull request merged**